src/components/DataTable/DataTable.story.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import type { Hst } from '@histoire/plugin-svelte';
+	import DataTable from './DataTable.svelte';
+	export let Hst: Hst;
+	let tableData = [
+		{
+			'': 'ครม.',
+			'ร่าง พ.ร.บ.': 'มีสิทธิเสนอร่าง',
+			'ออก พ.ร.ก.': 'มีสิทธิเสนอร่าง',
+			'ร่าง พ.ร.ป.': 'มีสิทธิเสนอร่าง',
+			'ร่างแก้ไขบทบัญญัติใน รธน.': 'มีสิทธิเสนอร่าง'
+		},
+		{
+			'': 'สส.',
+			'ร่าง พ.ร.บ.': '20 คน',
+			'ออก พ.ร.ก.': '-',
+			'ร่าง พ.ร.ป.': 'สส. ไม่น้อยกว่า 1 ใน 10',
+			'ร่างแก้ไขบทบัญญัติใน รธน.': 'สส. ไม่น้อยกว่า 1 ใน 5'
+		},
+		{
+			'': 'สมาชิกรัฐสภา',
+			'ร่าง พ.ร.บ.': '-',
+			'ออก พ.ร.ก.': '-',
+			'ร่าง พ.ร.ป.': '-',
+			'ร่างแก้ไขบทบัญญัติใน รธน.': 'สส. ไม่น้อยกว่า 1 ใน 5'
+		},
+		{
+			'': 'ประชาชนผู้มีสิทธิเลือกตั้ง',
+			'ร่าง พ.ร.บ.': '10,000 คน',
+			'ออก พ.ร.ก.': '-',
+			'ร่าง พ.ร.ป.': '-',
+			'ร่างแก้ไขบทบัญญัติใน รธน.': '50,000 คน'
+		}
+	];
+</script>
+
+<Hst.Story title="Table">
+	<div class="font-sans body-02"><DataTable {tableData} /></div>
```

Comment:
Adding control will help other people recognize the props more easily (it will show on the right panel of storybook). I suggest you take a look at [PoliticianProfile.story.svelte](src/components/PoliticianProfile/PoliticianProfile.story.svelte) to add control with `Hst.Json`